cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinContributor.java

```diff
@@ -26,13 +28,9 @@
 import nva.commons.core.paths.UriWrapper;
 
 @Data
-@Builder(
-    builderClassName = "CristinContributorBuilder",
-    toBuilder = true,
-    builderMethodName = "builder",
-    buildMethodName = "build",
-    setterPrefix = "with"
-)
+@Builder(builderClassName = "CristinContributorBuilder", toBuilder = true, builderMethodName = "builder",
+    buildMethodName = "build", setterPrefix = "with")
+@JsonIgnoreProperties({"identified_cristin_person"})
```

Comment:
You don't need the JsonIgnoreProperty now that you provide a class for it.

Reply:
Without this @JsonIgnoreProperty I am getting error message that says unknown not ignored field.

Reply:
Then your constructor is not taking it in properly.

Reply:
Daniel have added two new values, the one we map is "verified_person", the one we ignore is "identified_cristin_person".

Reply:
Ah, ok.